www/viewgame

```diff
@@ -1455,7 +1451,14 @@ function submitTime() {
         document.getElementById("timeSaveMsg").innerHTML="The total time must be at least one minute.";
         return;
     }
-    // Time is valid, as far as we can tell
+    // Time is valid, as far as we can tell.
+    // If the game isn't already on the user's list of played games, add it.
+    var played_it_checkbox = document.getElementById('ckBoxckPlayed');
+    if !played_it_checkbox.checked) {
```

Comment:
Syntax error, missing parenthesis. (This would be a good commit to try amending!)

Reply:
I had just fixed it